File: 2019-3-19/ForNancy/ForNancy/ViewController.m
//
//  ViewController.m
//  ForNancy
//
//  Created by admin on 2019/4/27.
//  Copyright © 2019 Sony. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UILabel *content;

@property(nonatomic,strong)UILabel *label;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    self.label=[[UILabel alloc] initWithFrame:CGRectMake(0, 200, 100, 20)];
    
    self.label.text=@"label";
    self.label.font=[UIFont systemFontOfSize:20.0];
    self.label.textAlignment=NSTextAlignmentCenter;
    
    [self.view addSubview:self.label];
}


@end

File: 2019-5-29(SWift)/Orientations/Orientations/ViewController.swift
//
//  ViewController.swift
//  Orientations
//
//  Created by admin on 2019/6/2.
//  Copyright © 2019 JY. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    private var _orientations = UIInterfaceOrientationMask.portrait.rawValue | UIInterfaceOrientationMask.landscapeLeft.rawValue
    
    override var supportedInterfaceOrientations: UIInterfaceOrientationMask{
        get{return UIInterfaceOrientationMask(rawValue: self._orientations)}
        set{self._orientations = newValue.rawValue}
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }
    
    
}

